runtime/src/app/app.ts

```diff
@@ -274,7 +274,8 @@ export async function hydrate_target(target: Target): Promise<{
 	branch?: Array<{ Component: ComponentConstructor, preload: (page) => Promise<any>, segment: string }>;
 }> {
 	const { route, page } = target;
-	const segments = page.path.split('/').filter(Boolean);
+	const segments = page.path === '/' ? ['index'] : page.path.split('/').filter(Boolean);
```

Comment:
So, this change will not allow us to have a route like `index`? Why not just pass the empty string `''` instead? It's basically what we have in url.